src/main/java/com/google/sps/servlets/BooksServlet.java

```diff
@@ -45,9 +46,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<String> terms = Arrays.asList(request.getParameterValues("key"));
     LinkedHashMap<String, List<Book>> booksMap = new LinkedHashMap<>();
     terms.forEach((term) -> {
-      String jsonString = getJsonStringForTerm(term);
-      List<Book> books = makeBooksList(jsonString);
-      booksMap.put(term, books);
+      try {
+        String jsonString = getJsonStringForTerm(term);
+        List<Book> books = makeBooksList(jsonString);
+        booksMap.put(term, books);
+      } catch(Exception e) {
```

Comment:
[Nit] Please put a space between 'catch' and '('.  

This applies here and in the other files for this PR.